requirements.txt

```diff
@@ -30,6 +31,7 @@ eth-rlp==0.1.2
 eth-tester==0.1.0b37
 eth-typing==2.0.0
 eth-utils==1.4.1
+ethpm==0.1.4a12
```

Comment:
This package is OK here - it does not interfere. (only dev installation is affected by `py-solc` issues.)